src/scoreExplain.c

```diff
@@ -0,0 +1,70 @@
+#include "scoreExplain.h"
+#include "rmalloc.h"
+
+struct RSScoreExplain{
+  char *str;
+  int numChildren;
+  RSScoreExplain **children;
```

Comment:
use the array type in `arr`

Reply:
kept same mechanism as in RSIndexResult & RSAggregateResult.
I think it is simple and straightforward.